data/kube-secondary-dns/secondarydns.yaml

```diff
@@ -73,7 +73,7 @@ spec:
     metadata:
       labels:
         k8s-app: secondary-dns
-        hco.kubevirt.io/allow-access-cluster-services: ""
+        allow-access-cluster-services: ""
```

Comment:
please see here about prefix that is nice to have
https://github.com/kubevirt/hyperconverged-cluster-operator/commit/a77e454da7d1200fa4d8007045cba64dfde66d9d

Reply:
i meant of course to use prefix of cnao,
but yes you are right
thanks